charts/dapr/README.md

```diff
@@ -172,67 +172,67 @@ The Helm chart has the follow configuration options that can be supplied:
-| `dapr_sidecar_injector.allowedServiceAccountsPrefixNames` | Comma-separated list of extra allowed service accounts. Each item in the list should be in the format of namespace:serviceaccount. To match service accounts by a common prefix, you can add an asterisk (`*`) at the end of the prefix. For instance, ns1*:sa2* will match any service account that starts with sa2, whose namespace starts with ns1. For example, it will match service accounts like sa21 and sa2223 in namespaces such as ns1, ns1dapr, and so on. | `""` |
-| `dapr_sidecar_injector.resources`                         | Value of `resources` attribute. Can be used to set memory/cpu resources/limits. See the section "Resource configuration" above. Defaults to empty                                                                                                                                                                                                                                                                                                                      | `{}` |
-| `dapr_sidecar_injector.debug.enabled`                     | Boolean value for enabling debug mode                                                                                                                                                                                                                                                                                                                                                                                                                                  | `{}` |
-| `dapr_sidecar_injector.kubeClusterDomain`                 | Domain for this kubernetes cluster. If not set, will auto-detect the cluster domain through the `/etc/resolv.conf` file `search domains` content.                                                                                                                                                                                                                                                                                                                      | `cluster.local` |
+| Parameter                                                 | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Default                                                                                                                                                                                                                   |
```

Comment:
Please don't change things that are unrelated in this README. Adding all this whitespace actually makes reading the README _harder_ when not using the rendered markdown...

Reply:
This diff makes the tables properly formatted and consistent. Some of the tables were properly aligned and the ones updated in this PR were not properly aligned, hence the diff. My IDE was complaining about the format and auto fixed it. If you have it locally it reads fine and the text is aligned well. I would think having everything properly aligned would lean towards being easier to digest.